frontend/components/tree/js/tree.component.ts

```diff
@@ -416,14 +416,39 @@ class TreeController implements ng.IController {
 		this.TreeService.updateModelState(node);
 	}
 
+	public selectAndCentreNode(node: any) {
+		if(node.toggleState !== "invisible") {
```

Comment:
For consistency space after if, you could also do early return here keep the code flatter